pom.xml

```diff
@@ -115,11 +115,11 @@
       -->
 
       <activemq-surefire-argline>-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-         -Dlogging.configuration=file:${activemq.basedir}/tests/config/logging.properties
-         -Djava.library.path=${activemq.basedir}/artemis-native/bin/ -Djgroups.bind_addr=localhost -Dorg.apache.activemq.artemis.api.core.UDPBroadcastEndpointFactory.localBindAddress=localhost
+         -Dlogging.configuration='file:${activemq.basedir}/tests/config/logging.properties'
+         -Djava.library.path='${activemq.basedir}/artemis-native/bin/' -Djgroups.bind_addr=localhost -Dorg.apache.activemq.artemis.api.core.UDPBroadcastEndpointFactory.localBindAddress=localhost
          -Djava.net.preferIPv4Stack=true
       </activemq-surefire-argline>
-      <activemq.basedir>${project.basedir}</activemq.basedir>
+      <activemq.basedir>${basedir}</activemq.basedir>
```

Comment:
There are some sub projects that will need to find files in reference to the base-dir... (e.g. the native library)... and we had issues with basedir in the past. Maybe it's a versioning thing?


Also this is used on Idea settings. When you run tests. it will pickup the location from this property and it is a bit tricky to get it working. I would need to test and see if it's ok.